### DISCUSSION TITLE
有什么一键更新所有mod的软件吗

### DISCUSSION THREAD

**Discussion by MotWnb** [Meloong-Git/PCL#2095] [提问] [answered]
如题

**Comment by WForst-Breeze** [comment, marked as answer]
可以先尝试隔壁 HMCL 启动器看看？

**Comment by gyc123456-1** [comment]
cuseforge客户端试试？

**Comment by MotWnb** [comment]
谢谢



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年06月23日 17:59 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [Hex-Dragon/PCL2] 有什么一键更新所有mod的软件吗 (Discussion #2095) |

可以先尝试隔壁 HMCL 启动器看看？
